easybuild/easyconfigs/m/MOOSE/MOOSE-2021-05-18-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,66 @@
+easyblock = 'Binary'
+
+name = 'MOOSE'
+# corresponds to commit f5fa9f0
+version = '2021-05-18'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://mooseframework.inl.gov'
+description = """The Multiphysics Object-Oriented Simulation Environment (MOOSE) is a finite-element, multiphysics
+framework primarily developed by Idaho National Laboratory"""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+toolchainopts = {'usempi': True}
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('libpng', '1.6.37'),
+    ('PETSc', '3.12.4', versionsuffix),
+    ('SLEPc', '3.12.2', versionsuffix),
+    ('VTK', '8.2.0', versionsuffix),
+    ('sympy', '1.5.1', versionsuffix),
+    ('ParMETIS', '4.0.3'),
+]
+
+sources = [{
+    'filename': SOURCE_TAR_GZ,
+    'git_config': {
+        'url': 'https://github.com/idaholab',
+        'repo_name': 'moose',
+        'tag': version,
+        'recursive': True,
+        'keep_git_dir': True,
+    },
+}]
+checksums = [None]
+
+extract_sources = True
+
+buildininstalldir = True
+
+# see https://mooseframework.inl.gov/getting_started/installation/hpc_install_moose.html
+# enable building libmesh with -march=native
+local_libmesg_configopts = "--disable-warnings --enable-march --with-mpi=$EBROOTMPI"
+install_cmd = "sed 's/--disable-warnings/%s/g' scripts/update_and_rebuild_libmesh.sh && " % local_libmesg_configopts
+# build vendored copy of libmesh
+install_cmd += 'export CXXFLAGS="$CXXFLAGS -fpermissive" && export MOOSE_JOBS="%(parallel)s V=1" && '
+install_cmd += "./scripts/update_and_rebuild_libmesh.sh --skip-submodule-update && "
+# build MOOSE itself
+install_cmd += "cd test && pwd && make -j %(parallel)s && make -j %(parallel)s hit && "
+# run tests
+install_cmd += "echo 'running tests' && export PYTHONPATH=%(installdir)s/moose/moosetools/contrib/hit:$PYTHONPATH && "
+# run tests, but tolerate failures
+install_cmd += "(python run_tests -j %(parallel)s || echo 'Some tests are failing!' >&2 ) "
```

Comment:
```suggestion
install_cmd += "(python run_tests -j %(parallel)s --max-fails 10000 || echo 'Some tests are failing!' >&2 ) "
```

Reply:
What does this option do exactly?

Reply:
Yes, but there are less than 10k tests (currently) and the default value for `--max-fails` is 50. This is why https://github.com/easybuilders/easybuild-easyconfigs/pull/13824#discussion_r695939182 stopped at 50 failures. This is probably no longer needed, as we are below 50 failures now - when I suggested this I did not know how many more tests there were and how many more failures we'd see.

Reply:
It doesn't hurt to still use this though, there's no negative side effect, makes sense to ensure we have a clear view on the test results